express/styles/styles.css

```diff
@@ -559,6 +559,10 @@ a:any-link {
   padding-bottom: 40px;
 }
 
+.section:has(.fragment) .content {
```

Comment:
is this actually necessary anymore? 
when I took a look at the template-x block which gets loaded in a fragment it is taking the full-width. I think Jingle's unwrap might only be necessary for our old project fortunately.